presto-tests/src/test/java/com/facebook/presto/tests/TestQueryTaskLimit.java

```diff
@@ -26,42 +28,55 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Future;
 
+import static com.facebook.presto.execution.QueryState.FAILED;
+import static com.facebook.presto.execution.QueryState.QUEUED;
+import static com.facebook.presto.execution.QueryState.RUNNING;
+import static com.facebook.presto.execution.TestQueryRunnerUtil.cancelQuery;
+import static com.facebook.presto.execution.TestQueryRunnerUtil.createQuery;
+import static com.facebook.presto.execution.TestQueryRunnerUtil.waitForQueryState;
+import static com.facebook.presto.execution.TestQueues.LONG_LASTING_QUERY;
+import static com.facebook.presto.execution.TestQueues.newSession;
 import static com.facebook.presto.spi.StandardErrorCode.QUERY_HAS_TOO_MANY_STAGES;
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 import static java.util.concurrent.Executors.newCachedThreadPool;
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
 
+@Test(singleThreaded = true)
 public class TestQueryTaskLimit
 {
     private ExecutorService executor;
+    private Session defaultSession;
 
     @BeforeClass
     public void setUp()
     {
         executor = newCachedThreadPool();
+        defaultSession = testSessionBuilder()
+                .setCatalog("tpch")
+                .setSchema("sf1000")
+                .build();
     }
 
     @AfterClass(alwaysRun = true)
     public void shutdown()
+            throws Exception
     {
         executor.shutdownNow();
+        assertTrue(executor.awaitTermination(10, SECONDS));
```

Comment:
Unrelated change?

Reply:
Not directly related to queueing test,  I created a separate PR to avoid confusion